src/main/resources/META-INF/reaktivity/mqtt.idl

```diff
@@ -88,16 +64,19 @@ scope mqtt
             uint32 messageExpiry;
```

Comment:
Is this absolute or relative, and what are the units?
I'm checking to see if the `uint32` field size is sufficient.